pkgs/by-name/co/cosmic-bg/package.nix

```diff
@@ -21,6 +22,14 @@ rustPlatform.buildRustPackage (finalAttrs: {
     hash = "sha256-4b4laUXTnAbdngLVh8/dD144m9QrGReSEjRZoNR6Iks=";
   };
 
+  patches = [
+  ];
+
+  postPatch = ''
+    substituteInPlace config/src/lib.rs data/v1/all \
+      --replace-fail '/usr/share/backgrounds/cosmic/orion_nebula_nasa_heic0601a.jpg' "${cosmic-wallpapers}/share/backgrounds/cosmic/orion_nebula_nasa_heic0601a.jpg"
```

Comment:
[nit] I would write this as follows, so that the line doesn't end up so long:

```sh
    substituteInPlace config/src/lib.rs data/v1/all --replace-fail \
      '/usr/share/backgrounds/cosmic/orion_nebula_nasa_heic0601a.jpg' \
      "${cosmic-wallpapers}/share/backgrounds/cosmic/orion_nebula_nasa_heic0601a.jpg"
```